addons/website_sale/static/src/website_builder/products_item_option.js

```diff
@@ -18,20 +18,36 @@ export class ProductsItemOption extends BaseOptionComponent {
             itemSize: this.props.itemSize,
         });
 
+        this.getProductsGridTable = () => {
```

Comment:
Convention: variables pointing to an element (or functions returning one) should be suffixed by `El`.
`Els` in case of multiple entries.

```suggestion
        this.getProductsGridTableEl = () => {
```